validation.py

```diff
@@ -1,35 +1,39 @@
 from nfldb_tables import NFLFrames
-from modeling import PositionNMFFactory
+from sklearn.metrics import mean_squared_error
+from factorize import nmf_all_positions, merge_factorizations_to_main_df, pred_from_factorized_skills
+from data_prep_tools import get_yr_until_wk, get_preds_to_make, positions_list
 
-def year_week_position_rmse(year, week, position, predictions, nfl_frames):
+def year_week_rmse(week_actuals_df, preds):
     '''
-    Input:  Int - year, Int - week to validate, Str - position predictions are for,
-            DataFrame - predictions against, NFLFrames
-    Output: RMSE between the predicted values for year-week and actual values from year-week
+    Input:  DataFrames with actual fanduel_points and DF with pred column.  
+            Both must have player_id
+    Output: RMSE between predicted and actual values from year-week
     '''
-    week_position_df = nfl_frames.get_year_week_frame(year, week).query('position == @position')
-    week_postion_predictions_df = week_position_df.merge(right=predictions,
-                                                         how='inner',
-                                                         on='player_id')
-    week_postion_predictions_df.eval('error = fanduel_points - prediction')
     
-    rmse =  ((week_postion_predictions_df.error.values ** 2) ** 0.5).mean()
```

Comment:
Here's, at least part, of the difference between our RMSE's. I was taking the mean at the wrong step...